### PR TITLE
Check the return value of Get Window Rect in the payload test

### PR DESCRIPTION
**Pull request by gsnedders**

### CODE REVIEW

**Comment by gsnedders** [comment]
(I suppose we could get rid of the rest of the asserts without any change in what we're testing.)

File: webdriver/tests/get_window_rect.py [hunk 1]
```diff
@@ -151,11 +151,10 @@ def test_payload(session):
...
     assert isinstance(response.body["value"], dict)
     value = response.body["value"]
-    assert "width" in value
-    assert "height" in value
-    assert "x" in value
-    assert "y" in value
-    assert isinstance(value["width"], int)
-    assert isinstance(value["height"], int)
-    assert isinstance(value["x"], int)
-    assert isinstance(value["y"], int)
```
Comment (andreastt) [inline webdriver/tests/get_window_rect.py]: I’m not entirely comfortable removing the explicit assertions of the data integrity, but I guess this is fine.

**Review by andreastt — APPROVED** [review]

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/21479)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/312024537)

File: webdriver/tests/get_window_rect.py (hunk 1 above)
Comment (gsnedders) [reply webdriver/tests/get_window_rect.py]: In case `window.screenX` doesn't return an integer, for example?

Comment (andreastt) [reply webdriver/tests/get_window_rect.py]: Yes, this has known to happen. But I think pytest will catch this when it does a deep dict comparison?

Comment (gsnedders) [reply webdriver/tests/get_window_rect.py]: Right, it does compare the entire dict. Obviously, if `window.screenX` and what Get Window Rect return are broken in the same way then it won't catch it now. So maybe we should have these?

Comment (andreastt) [reply webdriver/tests/get_window_rect.py]: That would be out of scope of the WebDriver tests, but I wouldn’t object to keeping the assertions around.

Comment (andreastt) [reply webdriver/tests/get_window_rect.py]: Just to give some context: Marionette used to return floats for these values.

Comment (gsnedders) [reply webdriver/tests/get_window_rect.py]: But if it did, then it wouldn't match `window.screenX`, no? (Assuming a conforming `window.screenX` implementation.)

Comment (andreastt) [reply webdriver/tests/get_window_rect.py]: Yes, I would assume so. As I said, happy to land this as-is.

**Pull request merged**